vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/post_transaction_gemini/post_transaction_sku_gemini.cc

```diff
@@ -66,7 +66,7 @@ type::Result PostTransactionGemini::CheckStatusCode(const int status_code) {
     return type::Result::LEDGER_ERROR;
   }
 
-  if (status_code != net::HTTP_CREATED) {
+  if (status_code != net::HTTP_CREATED && status_code != net::HTTP_OK) {
```

Comment:
HTTP `200` will be returned when the external transaction ID has already been posted to the order. We need to call this endpoint on retry so that the server can re-check the external transaction status.